cmd/krew/cmd/list.go

```diff
@@ -24,14 +24,92 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/printers"
 
 	"sigs.k8s.io/krew/pkg/installation"
 )
 
+type Entry struct {
```

Comment:
For `krew info`, Entry can be expanded...

```golang
	URI         string `yaml:"omitifempty" json:"omitifempty"`
	SHA256      string `yaml:"omitifempty" json:"omitifempty"`
	URL         string `yaml:"omitifempty" json:"omitifempty"`
	Description string `yaml:"omitifempty" json:"omitifempty"`
```

Alternatively, a new data structure.